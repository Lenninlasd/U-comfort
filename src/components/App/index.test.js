import React from 'react';
import { App } from './index';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('<App /> shallow rendering', () => {
  it('matches the snapshot', () => {
    const tree = shallow(<App />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
