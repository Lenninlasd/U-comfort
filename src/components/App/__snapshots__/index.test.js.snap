// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> shallow rendering matches the snapshot 1`] = `
<BrowserRouter>
  <div>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/equipment"
    />
  </div>
</BrowserRouter>
`;
